index.bs

```diff
@@ -3530,6 +3530,26 @@ operation. Likewise, clients can choose to produce a [=client extension output=]
 understand by encoding the [=authenticator extension output=] value into JSON, provided that the CBOR output uses only types
 present in JSON.
 
+When clients choose to pass through extensions they do not recognize,
+the JavaScript values in the [=client extension input=] values
+are converted to [=CBOR=] values in the [=authenticator extension input=] values.
+When the JavaScript input value is an [=%ArrayBuffer%=], it is converted to a [=CBOR=] byte array.
+When the JavaScript input value is a non-integer number, it is converted to a 64-bit CBOR floating point number.
+Otherwise, when the JavaScript input type corresponds to a JSON type, the conversion is done
+using the rules defined in Section 4.2 of [[RFC7049]] (Converting from JSON to CBOR),
+but operating on JavaScript inputs rather than JSON inputs.
+Once these conversions are done,
+canonicalization of the resulting [=CBOR=] MUST be performed using the [=CTAP2 canonical CBOR encoding form=].
+
+Likewise, when clients receive outputs from extensions they have passed through that they do not recognize,
+the [=CBOR=] values in the [=authenticator extension output=] values
+are converted to JavaScript values in the [=client extension output=] values.
+When the CBOR input value is a byte string, it is converted to a JavaScript [=%ArrayBuffer%=]
+(rather than a base64url-encoded string).
+Otherwise, when the CBOR input type corresponds to a JSON type, the conversion is done
+using the rules defined in Section 4.1 of [[RFC7049]] (Converting from CBOR to JSON),
+but producing JavaScript outputs rather than JSON outputs.
```

Comment:
s/ JavaScript outputs rather than JSON outputs / outputs of JavaScript type values rather than JSON-encoded outputs/   ...?